src/main/java/com/urswolfer/gerrit/client/rest/http/changes/RevisionApiRestClient.java

```diff
@@ -245,6 +245,11 @@ public SubmitType testSubmitType(TestSubmitRuleInput in) throws RestApiException
         return gerritRestClient.getGson().fromJson(jsonElement, new TypeToken<SubmitType>() {}.getType());
     }
 
+    public String description() throws RestApiException {
```

Comment:
can you please add `@Override`?

Reply:
Ops, I forgot to add it. Sorry. 
Thank you for pointing it out, I fixed it now.